### ISSUE TITLE
10-7959c: update copy - Signature 

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#83952]
As a user on VA.gov, I want to be able to sign when filling out the OHI form so that I can certify any changes to my other health insurance.

Page elements:
- breadcrumb
- header and subhead
- progress indicator
- footer

Pages (following file upload screen): 
- Form signature (beneficiary, sponsor, representative)
  - if beneficiary signs, then show Statement of truth where they fill out their name 
  - if sponsor, parent or representative signs, then show Statement of truth where the beneficiary full name is already populated 
    - sponsor, parent or representative must enter their name into the text field

Figma link ("Prep for staging review"): https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/10-7959c-Form?node-id=1155-73969&t=Y8cWvTGeyw2lKO6X-0

Acceptance criteria:
  - Given the user is on VA.gov and is filling out the OHI form
  - When the user gets to the applicant signature section
  - Then the user can include their signature as part of the other health insurance certification submission

**Comment by michaelclement** [comment]
Work is done in these two PRs ([frontend](https://github.com/department-of-veterans-affairs/vets-website/pull/30349) and [backend](https://github.com/department-of-veterans-affairs/vets-api/pull/17118)). Waiting on:
- [ ] Team approval
- [ ] Platform approval

**Comment by marywang2** [comment]
@michaelclement there are some updates to the [figma] copy (https://www.figma.com/design/sSPoS7R9HE6ZlxRESRy2ms/Other-Health-Insurance-Certification-(10-7959c)-Form?node-id=1155-73969&t=8jkV7WpYOH4EiGtC-0)

**Comment by michaelclement** [comment]
@marywang2 This should be ready for review. Thanks!

**Comment by michaelclement** [comment]
Final wording change is in[ this PR ](https://github.com/department-of-veterans-affairs/vets-website/pull/30390)and not on staging yet, but it only impacts one bullet point:
<img width="690" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkJ4QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7e444e29e3dc79d03f690591db5569601836e198/340477367-380e87f7-ed22-4040-91cc-9c1aa2dd2f84.png" alt="340477367-380e87f7-ed22-4040-91cc-9c1aa2dd2f84.png" />

**Comment by marywang2** [comment]
@michaelclement putting this on pause for now until we can incorporate the latest copy updates from CAIA into the figma. cc @jamiefiore

**Comment by michaelclement** [comment]
Latest work is here in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30624).

**Comment by michaelclement** [comment]
@baltes @marywang2 @jamiefiore Merged and should be visible on staging shortly - thanks!

**Comment by baltes** [comment]
LGTM. @jamiefiore Could you take a look and confirm?

**Comment by jamiefiore** [comment]
@baltes @michaelclement Design wise it looks great. I noticed that a error message pops up really quick before moving to the confirmation page when a third party signs the Statement of Truth though

**Comment by michaelclement** [comment]
Thanks @jamiefiore!  -  let me look into that error and follow up here in a bit. We may want to split that into a separate ticket.

**Comment by michaelclement** [comment]
Thanks @jamiefiore!  -  let me look into that error and follow up here in a bit. We may want to split that into a separate ticket.

**Comment by michaelclement** [comment]
@baltes @jamiefiore I looked into this a bit more and I think it warrants a separate bug ticket. I've created that [here](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/87686). If that works for y'all, I'd recommend closing this one out. Thanks!
